tests/harrisLUT/StaticTransferMatrix.csv

```diff
@@ -0,0 +1,6092 @@
+Variable Name, Static Transfer Matrix
```

Comment:
Is this the same file you added to the ts_config_mttcs? You could use the copy there instead of duplicating it here. I know it means you need to have ts_config_mttcs for testing, but that should be straightforward to provide no? I just worry about having duplicated copies of files, and this one in particular is quite big.

Reply:
We discussed this problem before. Right now, the mechanism to read the `ts_config_mttcs` is by using the `ts_salobj` to do the `start` command to get the details by the `config` object. This will not work for the unit test that will need to read the test files if we want to use the `conda` (to build the `conda` package) instead of `eups` to get the path of `ts_config_mttcs`.

This happens to the `ts_m2` unit tests as well (and this is why we have a copy of configuration files in the `tests/harrisLUT` in `ts_m2` for the unit tests and build of the `conda` package. I think we should come out a better way in the future for this problem (I do not like to have multiple copies as well, but this was the best solution we agreed before).

Thanks!